oggm/core/flowline.py

```diff
@@ -851,9 +852,25 @@ def step(self, dt):
             thick_stag[1:-1] = (thick[0:-1] + thick[1:]) / 2.
             thick_stag[[0, -1]] = thick[[0, -1]]
 
+            sf_func = None
+            sf = np.ones(thick.shape)  # Default shape factor is 1
+            if cfg.PARAMS['use_shape_factor_for_fluxbasedmodel'] == \
+                    'Adhikari' or \
+                    cfg.PARAMS['use_shape_factor_for_fluxbasedmodel'] == 'Nye':
+                sf_func = shape_factor_adhikari
+            elif cfg.PARAMS['use_shape_factor_for_fluxbasedmodel'] == 'Huss':
+                sf_func = shape_factor_huss
+
+            sf = sf_func(fl.widths_m, thick, fl.is_rectangular)
```

Comment:
Tests are broken, probably due to this line and probably therein by fl.is_rectangular, as this is not always defined. Any good ideas on how to fix? By default assume rectangular grid?